reVX/turbine_flicker/turbine_flicker.py

```diff
@@ -0,0 +1,563 @@
+# -*- coding: utf-8 -*-
+"""
+Turbine Flicker exclusions calculator
+"""
+from concurrent.futures import as_completed
+from hybrid.flicker.flicker_mismatch_grid import FlickerMismatch
```

Comment:
Do we want to have this as an import in the init statement and not in the requirements? How heavy of an install is the hybrid library?

Reply:
That is a very good question. From PIP its not to bad. Its not an easy install from Conda. I'm fine with making it be a conditional requirement just for Turbine/shadow flicker

Reply:
I dont have a strong opinion and will definitely defer to you. It would definitely be nice to be able to run flicker tests with actions. But if the hybrid install isn't something we're going to use much it might be nice as an optional requirement. Once again, totally going to defer to you.